src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private Random color = new Random();
+    private int index = new Random().nextInt(Color.values().length);
+
     public String getRandomColor() {
-        return null;
+        String colorBall = String.valueOf(Color.values()[index]);
```

Comment:
```suggestion
        String color = String.valueOf(Color.values()[index]);
```
мы можем использовать этот метод не тольок для цвета шарика, но и для других моделек